src/main/java/org/commonjava/util/gateway/AdminResource.java

```diff
@@ -18,6 +18,7 @@
 import io.smallrye.mutiny.Uni;
 import io.vertx.core.http.HttpServerRequest;
 import io.vertx.core.json.JsonObject;
+import org.commonjava.util.gateway.metrics.response.component.ResponseCount;
```

Comment:
@GOptimistic is the requestcount already being declared before?
I'm asking because when doing AOP you should use your own interface declaration for annotations doing specific tasks. From this annotation name to me seems its purpose is to count the requests number and not the response time

Reply:
Thanks for your advice. I will try your approach and if the final choice is the AOP, how about the ResponseTimeLogging for the annotation name?